src/main.rs

```diff
@@ -129,6 +129,7 @@ fn build(app: &gtk::Application, opts: &Options) {
     ui_opts.set_cmdline_external(!opts.disable_ext_cmdline);
 
     ui_opts.set_wildmenu_external(true);
+    ui_opts.set_messages_external(true);
```

Comment:
Final version should have default false with option to enable (unless it gets really nice).